ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -14,12 +14,12 @@
 
 import unittest
 
-from flask import Flask, request
+import flask
```

Comment:
Why change this in this PR? Just adds more diff lines.

Reply:
Apparently I missed a `# FIXME Find out why 3 test cases of py37-test-ext-flask fail if I use from flask import Flask, request` here :sweat_smile: